File: distanceBetweenTwoPoints.go
package main
import (
	"fmt"
	"math"
)

func square(number float64) float64 {
	return number * number;
}

func main() {
	var x1, x2, y1, y2 float64;
	var diffDeXQuad, diffDeYQuad, res float64;

	fmt.Scanln(&x1, &y1);
	fmt.Scanln(&x2, &y2);

	diffDeXQuad = square(x2 - x1);
	diffDeYQuad = square(y2 - y1);

	res = math.Sqrt(diffDeXQuad + diffDeYQuad);

	fmt.Printf("%.4f\n", res)
}

File: distanceBetweenTwoPoints.c
#include <stdio.h>
#include <math.h>

int main() {
    float x1, y1, x2, y2;
    float diffDeXQuad, diffDeYQuad, res;

    scanf("%f %f", &x1, &y1);
    scanf("%f %f", &x2, &y2);

    diffDeXQuad = pow((x2 - x1), 2);
    diffDeYQuad = pow((y2 - y1), 2);

    res = sqrt(diffDeXQuad + diffDeYQuad);

    printf("%.4f\n", res);
}
